web/packages/teleterm/src/mainProcess/windowsManager.ts

```diff
@@ -486,7 +486,7 @@ export class WindowsManager {
         ? 'Keep Teleport Connect running in the menu bar?'
         : 'Keep Teleport Connect running in the system tray?',
       detail:
-        'VNet and connections to databases, kube clusters and apps will remain active.',
+        'VNet and connections to databases, Kubernetes clusters and apps will remain active.',
```

Comment:
```suggestion
        'VNet and connections to databases, Kubernetes clusters, and apps will remain active.',
```

One of the cases where I think the Oxford comma helps specify that we mean "connections to apps" and not just "apps".